src/actions/vstudio/vs2010_vcxproj.lua

```diff
@@ -1376,14 +1376,21 @@
 
 	function m.generateDebugInformation(cfg)
 		local lookup = {}
-		if _ACTION >= "vs2015" then
+		if _ACTION >= "vs2017" then
```

Comment:
I think this chance is good, and we can merge this as is....

I do however like to see someone create a PR to move this table into the action definition (vs2010/vs2012/vs2013/vs2015/vs2017.lua)